src/wp-includes/class-wp-term.php

```diff
@@ -99,6 +99,14 @@ final class WP_Term {
 	 */
 	public $filter = 'raw';
 
+	/**
+	 * Stores the sanitized term data.
+	 *
+	 * @since 6.6.0
+	 * @var stdClass|null
+	 */
+	protected $data;
```

Comment:
Should the `$data` property be private or protected?
Making it protected is probably fine, as I see no reason to make it private.